modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/thunks/previousPage.es.js

```diff
@@ -42,11 +43,16 @@ export default function previousPage({
 		}).then((evaluatedPages) => {
 			let previousActivePageIndex = activePage;
 
-			for (let i = activePage - 1; i > -1; i--) {
-				if (evaluatedPages[i].enabled) {
-					previousActivePageIndex = i;
+			if (typeof selectedPage === 'number') {
```

Comment:
In this condition, are you doing this to verify that the `selectedPage` isn't null? If that's the case wouldn't be more clear to put if `(selectedPage !== null)` instead? Apologies if I understood this wrong!

Reply:
She explained to me that's because the index can be 0 and also `undefined`, so this is indeed a good way to make the verification. Thanks Aline!